### ISSUE TITLE
Generated PHP source causes fatal error with double values

### ISSUE THREAD

**Issue by jfiala** [swagger-api/swagger-codegen#622]
develop-2.0 / php, Swagger 2.0 API-docs

api-docs (Swagger 2.0):
"longitude":{"required":false,"format":"double","type":"number"

full api-docs
```
{"apiVersion":"1.0","apis":[{"description":"saveUser","operations":[{"method":"POST","summary":"create or update a user name by id","notes":"saveUser","nickname":"saveUser","produces":["*/*"],"consumes":["application/json"],"parameters":[{"allowMultiple":false,"defaultValue":"","description":"id","name":"id","paramType":"query","format":"int64","type":"integer","required":true},{"allowMultiple":false,"defaultValue":"","description":"name","name":"name","paramType":"query","type":"string","required":true}],"responseMessages":[{"code":200,"message":"User has been updated","responseModel":"User"}],"deprecated":"false","type":"User"}],"path":"/user"},{"description":"getUser","operations":[{"method":"GET","summary":"read User by ID","notes":"pass ID to read user","nickname":"getUser","produces":["*/*"],"consumes":["application/json"],"parameters":[{"allowMultiple":false,"defaultValue":"","description":"id","name":"id","paramType":"query","format":"int64","type":"integer","required":true}],"responseMessages":[{"code":200,"message":"Successful retrieval of user detail","responseModel":"User"},{"code":404,"message":"User not found","responseModel":"Void"}],"deprecated":"false","type":"User"}],"path":"/user"}],"basePath":"/","consumes":["application/json"],"models":{"User":{"description":"User object","id":"User","properties":{"id":{"description":"ID of the user","required":true,"format":"int64","type":"integer"},"locations":{"items":{"type":"Location"},"required":false,"type":"array"},"name":{"description":"Name of the user","required":true,"type":"string"},"categories":{"items":{"type":"Category"},"required":false,"type":"array"},"photo":{"items":{"format":"byte","type":"string"},"required":false,"type":"array"}}},"Category":{"description":"","id":"Category","properties":{"id":{"required":false,"format":"int64","type":"integer"},"name":{"required":false,"type":"string"}}},"Location":{"description":"","id":"Location","properties":{"zip":{"required":false,"type":"string"},"street":{"required":false,"type":"string"},"longitude":{"required":false,"format":"double","type":"number"},"latitude":{"required":false,"format":"double","type":"number"},"country":{"required":false,"type":"string"},"city":{"required":false,"type":"string"}}}},"produces":["*/*"],"resourcePath":"/user","swaggerVersion":"1.2"}
```

generated source:
```php

namespace SwaggerPetstore\models;

use \ArrayAccess;

class Location implements ArrayAccess {
  static $swaggerTypes = array(
      'street' => 'string',
      'zip' => 'string',
      'city' => 'string',
      'country' => 'string',
      'latitude' => 'double',
      'longitude' => 'double'
  );

  static $attributeMap = array(
      'street' => 'street',
      'zip' => 'zip',
      'city' => 'city',
      'country' => 'country',
      'latitude' => 'latitude',
      'longitude' => 'longitude'
  );

  
  public $street; /* string */
  public $zip; /* string */
  public $city; /* string */
  public $country; /* string */
  public $latitude; /* double */
  public $longitude; /* double */

  public function __construct(array $data = null) {
    $this->street = $data["street"];
    $this->zip = $data["zip"];
    $this->city = $data["city"];
    $this->country = $data["country"];
    $this->latitude = $data["latitude"];
    $this->longitude = $data["longitude"];
  }

  public function offsetExists($offset) {
    return isset($this->$offset);
  }

  public function offsetGet($offset) {
    return $this->$offset;
  }

  public function offsetSet($offset, $value) {
    $this->$offset = $value;
  }

  public function offsetUnset($offset) {
    unset($this->$offset);
  }
}

```

After calling the API, I get:
PHP Fatal error:  Class 'SwaggerPetstore\models\double' not found in C:\projects\swagger-spring-demo\user-rest-service-1
.0.2-client-php-codegen-develop-2.0\SwaggerPetstore-php\lib\APIClient.php on line 290
PHP Stack trace:

Can be reproduced using REST service:
https://github.com/jfiala/swagger-spring-demo/tree/master/user-rest-service-2.0.0 

Full generated client code:
https://github.com/jfiala/swagger-spring-demo/tree/master/user-rest-service-1.0.2-client-php-codegen-develop-2.0

**Comment by wing328** [comment]
Submitted #623 to fix the issue

**Comment by wing328** [comment]
I think this can be closed as #623 has been merged into develop_2.0

**Comment by jfiala** [comment]
Yes, perfect!
